src/main/resources/db/V8__create_hendelser_table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE hendelser (
+    hendelse_id           INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
Siden vi _stort sett_ bruker UUID-er som primærnøkler så kunne kanskje denne også vært en UUID for å bevare en slags standard, men det blir litt hipp som happ.

Reply:
Info om hvordan man lager en UUID-generator for Postgres: https://www.postgresqltutorial.com/postgresql-uuid/

Reply:
enig :)